code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm

```diff
@@ -81,6 +81,37 @@
 	result = /obj/item/reagent_containers/food/snacks/sashimi
 	subcategory = CAT_MEAT
 
+/datum/crafting_recipe/food/sushie_basic
+	name = "Funa Hosomaki"
+	reqs = list(
+		/datum/reagent/consumable/soysauce = 3,
+		/obj/item/reagent_containers/food/snacks/salad/ricebowl = 1,
+		/obj/item/reagent_containers/food/snacks/carpmeat = 2,
+		/obj/item/reagent_containers/food/snacks/grown/kudzupod = 3,
+	)
+	result = /obj/item/reagent_containers/food/snacks/sushie_basic
+	subcategory = CAT_MEAT
+
+/datum/crafting_recipe/food/sushie_adv
+	name = "Funa Nigiri"
+	reqs = list(
+		/datum/reagent/consumable/soysauce = 5,
+		/obj/item/reagent_containers/food/snacks/salad/ricebowl = 1,
+		/obj/item/reagent_containers/food/snacks/carpmeat = 1
+	)
+	result = /obj/item/reagent_containers/food/snacks/sushie_adv
+	subcategory = CAT_MEAT
+
+/datum/crafting_recipe/food/sushie_pro
+	name = "Well made Funa Nigiri"
+	reqs = list(
+		/datum/reagent/consumable/soysauce = 10,
+		/obj/item/reagent_containers/food/snacks/salad/ricebowl = 2,
+		/obj/item/reagent_containers/food/snacks/carpmeat = 5
```

Comment:
5 fillets??

Reply:
Yep Yep!
You taking the core of each one and seamlessly layering them for the best and freshens tastes as well as soaking them in a thin layer of soy sauce!

Reply:
i agree with pooj